tests/integration/test_rest.py

```diff
@@ -365,6 +379,17 @@ async def test_partitions(rest_async_client, admin_client, producer):
     res = await rest_async_client.get("/topics/fooo/partitions")
     assert res.status_code == 404
     assert res.json()["error_code"] == 40401
+
+    # Fill cache
+    res = await rest_async_client.get("/brokers")
```

Comment:
Should the metadata cache tests be separate set of tests, possibly using the `topic_details` API?